exercises/exercise-1.md

```diff
@@ -3,3 +3,22 @@ layout: guide
 ---
 
 # Exercise 1
+
+1. **Register an account on [developer.mbed.org](https://developer.mbed.org) and associate your dev board.**
+There is a [step-by-step guide available in the guides section](guides/mbed-org.html).
+
+1. **Import the location puck in the [online compiler](https://developer.mbed.org/compiler/).**
```

Comment:
Will it be obvious what the location puck is?
I guess how to find it will be explained in the import guide...